libraries/datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Constants.java

```diff
@@ -247,6 +251,9 @@ private SW360Constants() {
         SRC_ATTACHMENT_UPLOADER_EMAIL = props.getProperty("source.code.attachment.uploader.email", "");
         SRC_ATTACHMENT_DOWNLOAD_LOCATION = props.getProperty("src.attachment.download.location", "");
         PREFERRED_CLEARING_DATE_LIMIT =  props.getProperty("preferred.clearing.date.limit","");
+        MAIL_REQUEST_FOR_PROJECT_REPORT = Boolean.parseBoolean(props.getProperty("send.project.spreadsheet.export.to.mail.enabled", "false"));
+        MAIL_REQUEST_FOR_COMPONENT_REPORT = Boolean.parseBoolean(props.getProperty("send.component.spreadsheet.export.to.mail.enabled", "false"));
+        VCS_HOSTS = props.getProperty("vcs.hosts", "");
```

Comment:
The properties have been moved to the DB in #2815. Please rebase the PR and move the new property `vcs.hosts` to the container as well.

See #3191 or #3177 for reference implementation.